test/functional/tools/mulled_example_multi_2.xml

```diff
@@ -0,0 +1,79 @@
+<!-- same as mulled_example_multi_1, but requirements use builds
+     used to show that this does not yield the same mulled v2 hash,
+     i.e. the build is considered as part of version -->
+<tool id="mulled_example_multi_2" name="mulled_example_multi_2" version="0.1.0" license="AFL-3.0">
```

Comment:
The unit tests cover this already, I would prefer that we don't add this. In the end this is just a bug in mulled_build_tool.py which we don't use in Galaxy/

Reply:
> The unit tests cover this already, I would prefer that we don't add this. In the end this is just a bug in mulled_build_tool.py which we don't use in Galaxy.

I can remove it. The idea was to make sure that the parser really gives a version string with the build (maybe this test should be added somewhere) .. but we would realize very quickly if something goes wrong here anyway.

Reply:
Removed the test.